v2/cmd/asoctl/cmd/import_azure_resource.go

```diff
@@ -129,9 +129,9 @@ func importAzureResource(
 	}
 
 	done := make(chan struct{}) // signal that we're done
-	pb := progress.NewProgressBar("Import Azure Resources", progressBar, done)
+	pb := importreporter.NewBar("Import Azure Resources", progressBar, done)
 
-	importer := importing.NewResourceImporter(api.CreateScheme(), client, log, pb)
+	importer := importresources.NewResourceImporter(api.CreateScheme(), client, log, pb)
```

Comment:
minor: we could just make this `importresources.New` if we wanted to avoid repeating ourselves? Or maybe `NewImporter` (which is a bit of repetition still but less than we had before)

Reply:
Done.